panel/browser-panel.cpp

```diff
@@ -237,10 +239,11 @@ void QCefWidgetInternal::Init()
 		size *= devicePixelRatio();
 #endif
 		RECT rc = {0, 0, size.width(), size.height()};
-		windowInfo.SetAsChild((HWND)id, rc);
-#elif __APPLE__
-		windowInfo.SetAsChild((CefWindowHandle)id, 0, 0, size.width(),
-				      size.height());
+		windowInfo.SetAsChild((HWND)this->window->winId(), rc);
+#else
+		CefRect rc = {0, 0, size.width(), size.height()};
```

Comment:
What is the difference between RECT and CefRect?  Is this a platform specific difference?

Reply:
My guess based on the documentation is that CefRect is a platform-agnostic method.

https://magpcss.org/ceforum/apidocs3/projects/(default)/CefWindowInfo.html#SetAsChild(cef_window_handle_t,RECT)

Reply:
Changed to CefRect, still needs confirmation it still compiles on Windows.

Reply:
My hope is that if we use CefRect everywhere, it's more uniform and less code in ifdefs.

Reply:
Nope. It has to use RECT on Windows.

Reply:
That's unfortunate.  Oh well.